projects/example-app/src/app/auth/reducers/index.ts

```diff
@@ -17,10 +16,7 @@ export interface State extends fromRoot.State {
   auth: AuthState;
 }
 
-export const reducers: ActionReducerMap<
-  AuthState,
-  AuthApiActions.AuthApiActionsUnion
-> = {
+export const reducers: ActionReducerMap<AuthState, any> = {
```

Comment:
What's the reason of this change?

Reply:
Fixes typescript warning error due to action union, i believe its the reason the other `ActionReducerMap`'s are declared this way within the example app

Reply:
Yea, you're right.
I'm just not seeing these errors in this file 😅